Firestore/core/src/firebase/firestore/core/sync_engine.cc

```diff
@@ -281,13 +281,19 @@ void SyncEngine::HandleRejectedListen(TargetId target_id, Status error) {
     // store's invariants with another method.
     NoDocument doc(limbo_key, SnapshotVersion::None(),
                    /* has_committed_mutations= */ false);
-    DocumentKeySet limbo_documents = DocumentKeySet{limbo_key};
-    RemoteEvent event{SnapshotVersion::None(), /*target_changes=*/{},
-                      /*target_mismatches=*/{},
-                      /*document_updates=*/{{limbo_key, doc}},
+
+    // Explicitly instantiate these to work around a spurious "chosen
+    // constructor is explicit in copy-initialization" error in the RemoteEvent
+    // constructor on GCC 4.9.
+    DocumentKeySet limbo_documents{limbo_key};
+    RemoteEvent::TargetChangeMap target_changes;
+    RemoteEvent::TargetSet target_mismatches;
+    RemoteEvent::DocumentUpdateMap document_updates{{limbo_key, doc}};
+
+    RemoteEvent event{SnapshotVersion::None(), std::move(target_changes),
+                      std::move(target_mismatches), std::move(document_updates),
```

Comment:
Optional: you can instantiate these on the fly if you'd like:
```
RemoteEvent event{SnapshotVersion::None(), RemoteEvent::TargetChangeMap{},
  // ...and so on...
```
OTOH, perhaps this change would make the line too verbose, so feel free to keep as is.

Reply:
Left as-is.